docker/onadata-uwsgi/Dockerfile

```diff
@@ -1,4 +1,26 @@
+FROM python:3.6 as intermediate
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV PYTHONUNBUFFERED 1
+
+ARG optional_packages
+# Optional Argument; Used in cases where the optional
+# packages require some sort of authentication i.e Private repositories on github
+ARG ssh_private_key
+
+# Create SSH Key file
+RUN mkdir /root/.ssh/
+RUN echo "${ssh_private_key}" > /root/.ssh/id_rsa && chmod 600 /root/.ssh/id_rsa
+
+# Add github to accepted domains
+RUN touch /root/.ssh/known_hosts
+RUN ssh-keyscan github.com > /root/.ssh/known_hosts
+
+# Install optional requirements
+RUN if [ ! -z "$optional_packages" ]; then pip install ${optional_packages} ; fi
```

Comment:
Codacy found an issue: [Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6384747)